kubelet/datadog_checks/kubelet/kubelet.py

```diff
@@ -188,7 +188,9 @@ def retrieve_pod_list(self):
                 # Sanitize input: if no pod are running, 'items' is a NoneObject
                 pod_list['items'] = []
             return pod_list
-        except Exception:
+        except Exception as e:
+            self.log.debug('failed to retrieve pod list from the kubelet at %s : %s'
```

Comment:
I think an integration warning is legit if we cannot retrieve the podlist, instead of just a debug line. WDYT?

Reply:
ok, most failure cases (network / auth), will make the agent6 kubeutil fail anyway